Source/RapyutaSimulationPlugins/Private/RRROS2GameMode.cpp

```diff
@@ -60,5 +80,9 @@ void ARRROS2GameMode::InitROS2()
 
     // Simulation state
     SimulationState = currentWorld->SpawnActor<ASimulationState>();
-    SimulationState->Init(ROS2Node);
+//    ASimulationStateData* SimulationStateData = GetWorld() != NULL ? GetWorld()->GetGameState<ASimulationStateData>() : NULL;
+    SimulationStateData = currentWorld->SpawnActor<ASimulationStateData>();
```

Comment:
Does this actually need to be an actor? It's quite heavyweight if it's just a data storage container. If it needs to be serialized by Unreal a UObject derived class would be better, I think.